recipes/wasmedge/all/conandata.yml

```diff
@@ -1,4 +1,45 @@
 sources:
+  "0.14.0":
+    Windows:
+      "x86_64":
+        Visual Studio:
+          - url: "https://github.com/WasmEdge/WasmEdge/releases/download/0.14.0/WasmEdge-0.14.0-windows.zip"
+            sha256: "cd0dd57df8f3e8cb7ef0e57d2d09f1740093c2179fad933b270979bafe8295a1"
+          - url: "https://raw.githubusercontent.com/WasmEdge/WasmEdge/0.14.0/LICENSE"
+            sha256: "c71d239df91726fc519c6eb72d318ec65820627232b2f796219e87dcf35d0ab4"
+    Linux:
+      "x86_64":
+        "gcc":
+          - url: "https://github.com/WasmEdge/WasmEdge/releases/download/0.14.0/WasmEdge-0.14.0-manylinux2014_x86_64.tar.gz"
+            sha256: "73b3892f94c143dc09d53415c6848bb8e87206a1f614fd0edfc89957a0f1b027"
+          - url: "https://raw.githubusercontent.com/WasmEdge/WasmEdge/0.14.0/LICENSE"
+            sha256: "c71d239df91726fc519c6eb72d318ec65820627232b2f796219e87dcf35d0ab4"
+      "armv8":
+        "gcc":
+          - url: "https://github.com/WasmEdge/WasmEdge/releases/download/0.14.0/WasmEdge-0.14.0-manylinux2014_aarch64.tar.gz"
+            sha256: "6136c42066cdd9a96170285af2613dc00f262f5758a03d7afb3ab12a36363c79"
+          - url: "https://raw.githubusercontent.com/WasmEdge/WasmEdge/0.11.1/LICENSE"
```

Comment:
Probably not worth re-triggering a build for it since it is the exact same license file, but the version wasn't adjusted here.